enhancements/host-network-configuration.md

```diff
@@ -0,0 +1,398 @@
+static IP instead of relying on DHCP to be present on this VLAN.
+
+In this case I used a VLAN ID of 20, and the NIC is ens3.
+
+```
+[connection]
+id=vlan-vlan20
+type=vlan
+interface-name=vlan20
+autoconnect=true
+[ipv4]
+method=manual
+addresses=172.5.0.2/24
+gateway=172.5.0.1
+[vlan]
+parent=ens3
+id=20
+```
+
+These contents must then be base64 encoded and placed into a `MachineConfig`
+resource. Here's the resource I used:
+
+```
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 00-ocs-vlan
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2Nvbm5lY3Rpb25dCmlkPXZsYW4tdmxhbjIwCnR5cGU9dmxhbgppbnRlcmZhY2UtbmFtZT12bGFuMjAKYXV0b2Nvbm5lY3Q9dHJ1ZQpbaXB2NF0KbWV0aG9kPW1hbnVhbAphZGRyZXNzZXM9MTcyLjUuMC4yLzI0CmdhdGV3YXk9MTcyLjUuMC4xClt2bGFuXQpwYXJlbnQ9ZW5zMwppZD0yMAo=
+        filesystem: root
+        mode: 0600
+        path: /etc/NetworkManager/system-connections/ocs-vlan.conf
+
+```
+
+This resource can then be applied to a running cluster.
+
+Note that it could also be dropped in the `openshift/` directory of installer
+generated manifests if you’d like to have the installer apply a similar
+manifest at install time.  This works fine as long as you don’t need these
+network customizations applied before the host can reach the Machine Config
+Server.
+
+### RHCOS live ISO Environment
+
+As of OCP 4.5, [RHCOS includes a new live ISO/PXE ramdisk
+environment](https://github.com/openshift/enhancements/pull/210) for installing
+RHCOS.  This environment is used to run
+[coreos-installer](https://github.com/coreos/coreos-installer/). Following that
+work, [RHCOS will include the ability to run a text-based UI to configure
+networking](https://github.com/openshift/enhancements/pull/291) in the ramdisk:
+`nmtui`.  This installation process also provides the ability to provide
+networking configuration via Ignition if the administrator does not want to
+follow an interactive configuration process.
+
+These enhancements provide some excellent improvements to early network
+configuration capabilities.  However, it is currently most beneficial to the
+UPI bare metal flow.  The IPI bare metal flow will need to do more work to
+figure out how to make use of these tools.  Bare Metal IPI makes use of a
+different deployment ramdisk that is part of the Ironic project.  While
+conceptually similar, it has some significant differences in implementation.
+Future integration work may provide better alignment between the Bare Metal IPI
+tooling and the RHCOS installer.
+
+### SR-IOV Operator
+
+Some clusters may want to allocate SR-IOV devices to workloads for a secondary
+network interface with high performance.  These devices are managed by the
+[SR-IOV Operator[(https://github.com/openshift/sriov-network-operator).
```

Comment:
close bracket is backwards.

also this is pretty vague compared to everything else in the doc

Reply:
Fixed the bracket.  I haven't added more text yet.  I honestly don't know much of anything about the SR-IOV operator, so that's why it's vague.  :-)  I need to look at it more closely!